docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -1398,6 +1401,12 @@ Notes:
 - The `retry` block is prioritized over legacy retry fields (`retryable_errors`, `retry_max_attempts`, `retry_sleep_interval_sec`).
 - Conditional logic can be used within `ignorable_errors` to enable or disable rules dynamically.
 
+Evaluation Order:
+
+When processing errors, the **ignore** blocks are evaluated first. If an error matches an ignore rule, it is suppressed
```

Comment:
Make sure to add a note that the first matched pattern is used. If multiple conflicting patterns exist for the same block, any that match after the first one will be ignored.